examples/react-native/storybook/stories/Button.stories.tsx

```diff
@@ -4,8 +4,14 @@ import { action } from '@storybook/addon-actions';
 import { text } from '@storybook/addon-knobs';
 import { storiesOf } from '@storybook/react-native';
 import { Button } from '@aws-amplify/ui-react-native/dist/primitives';
+import { Screen } from '../ui';
 
 storiesOf('Button', module)
+  .addDecorator((Story: any) => (
+    <Screen>
+      <Story />
+    </Screen>
```

Comment:
We can reduce repetition by adding the decorator globally in the `storybook/index.tsx file`:

```
addDecorator(storyFn => <Screen>{storyFn()}</Screen>);
```